appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/RecoveryManager.java

```diff
@@ -181,6 +182,15 @@
     static void initialise() {
 
         // If already initialised, return immediately.
+        
+        String recoveryResynchTimeoutS = System.getProperty("fish.payara.jts.RecoveryResynchTimeout");
```

Comment:
it's easier just to call `Integer.getInteger()` - it will do the parsing and validation for you.